.github/workflows/ci.yml

```diff
@@ -21,7 +21,8 @@ jobs:
 
             - name: Install package and its dependencies
               run: |
-                  pip install .[dev]
+                  pip install .[dev,opencl]
+                  sudo apt install pocl-opencl-icd
```

Comment:
this is required or optional dependence? 
(sorry, I don't understand these things)

Reply:
This is just to modify how the automatic tests run here on Github after a push. It adds the necessary dependencies for testing the OpenCL code.